packages/frontend/src/components/common/Dashboard/DashboardRefreshButton.tsx

```diff
@@ -55,12 +61,16 @@ export const DashboardRefreshButton = () => {
         refreshInterval ? refreshInterval * 1000 * 60 : 0,
     );
 
+    useEffect(() => {
+        onIntervalChange(refreshInterval);
+    }, [onIntervalChange, refreshInterval]);
+
```

Comment:
I think we could do this on the `Menu.Item`s so it's clearer, what do you think?